### PR TITLE
fix lint rawnings in unitTestExample

### PR DESCRIPTION
**Pull request by zaki50**
@realm/java

### CODE REVIEW

**Comment by beeender** [comment]
:+1:

File: examples/unitTestExample/lint.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<lint>
+    <issue id="AllowBackup" severity="ignore" />
+    <issue id="IconLauncherShape" severity="ignore" />
+    <issue id="IconMissingDensityFolder" severity="ignore" />
+    <issue id="GoogleAppIndexingWarning" severity="ignore" />
+    <issue id="PrngFix" severity="ignore" />
```
Comment (nhachicha) [inline examples/unitTestExample/lint.xml]: remove please

File: examples/unitTestExample/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,9 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
 
     <string name="app_name">Unit Test Example</string>
-    <string name="hello_world">Hello world!</string>
-    <string name="action_settings">Settings</string>
     <string name="clean_up">Clean Up</string>
+    <string name="label" tools:ignore="TypographyEllipsis">Status Output...</string>
```
Comment (nhachicha) [inline examples/unitTestExample/src/main/res/values/strings.xml]: replace `...` with ellipsis

Comment (zaki50) [inline examples/unitTestExample/src/main/res/values/strings.xml]: done

File: examples/unitTestExample/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,9 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
```
Comment (cmelchior) [inline examples/unitTestExample/src/main/res/values/strings.xml]: We shouldn't need the tools ?

**Comment by cmelchior** [comment]
apart from small comment :+1:

**Pull request merged**